templates/studygroups/facilitator_feedback_done.html

```diff
@@ -0,0 +1,21 @@
+{% extends 'base.html' %}
+{% load static %}
+{% load i18n %}
+
+{% block content %}
+  <div class="container feedback-done">
+    <section class="inner-container centered col-xs-12 col-md-8">
+      <div class="centered">
+        <img src="{% static "images/thankyou.gif" %}" alt="Thank you!" style="max-height: 300px; max-width: 100%; width: auto;" />
+      </div>
+      <h2>{% trans "Thanks for participating! We hope to see you facilitate again soon." %}</h2>
+      <p>If you’re talking about your learning circle on social media, please mention <a href="https://twitter.com/p2pu">@p2pu</a> and <a href="https://twitter.com/hashtag/learningcircles">#learningcircles</a> so we can follow your journey.</p><br>
+      <div class="actions">
+        <a href="https://groups.google.com/a/p2pu.org/forum/#!forum/community" class="p2pu-btn blue">Join the community</a>
+        <a href="/http://p2pu.us2.list-manage.com/subscribe/post?u=f080ec78070523d59ef613eda&id=06e9c4ece8" class="p2pu-btn yellow">Subscribe</a>
+        <a href="https://shop.spreadshirt.com/p2pu/" class="p2pu-btn green">P2PU shop</a>
+        <a href="/en/facilitator/" class="p2pu-btn orange">Go to my dashboard</a>
```

Comment:
@s-kennedy rather than hardcode the URL, use `templates/studygroups/facilitator_feedback_done.html`